library/src/helpers/marked.ts

```diff
@@ -15,7 +16,7 @@ hljs.registerLanguage('yaml', yaml);
 import bash from 'highlight.js/lib/languages/bash';
 hljs.registerLanguage('bash', bash);
 
```

Comment:
Globally inline everything, just as the website does it:

```suggestion
let mermaidInitialized = false;
initializeMermaid();
function initializeMermaid() {
  if (mermaidInitialized) {
    return;
  }

  mermaidInitialized = true;
  mermaid.initialize({
    startOnLoad: true,
    theme: "base",
    securityLevel: "loose",
    themeCSS: ``,
    themeVariables: {
      primaryColor: "#EDFAFF",
      primaryBorderColor: "#47BCEE",
      secondaryColor: "#F4EFFC",
      secondaryBorderColor: "#875AE2",
      fontFamily: "Inter, sans-serif",
      fontSize: "18px",
      primaryTextColor: "#242929",
      tertiaryColor: "#F7F9FA",
      tertiaryBorderColor: "#BFC6C7"
    }
  });
}
```

This will most likely remove the error you have, about re-exporting the initializeMermaidOnce.